index.d.ts

```diff
@@ -38,6 +38,17 @@ declare namespace execa {
 		*/
 		readonly localDir?: string;
 
+		/**
+		Path to the Node.js executable to use in child processes.
+
+		This can be either an absolute path or a path relative to the `cwd` option.
+
+		Requires `preferLocal` to be `true`.
```

Comment:
Why does it require `preferLocal` to be `true`? Maybe include a short reasoning here.

Reply:
It's because the feature is handled by `npm-run-path` which is run only when the `preferLocal` option is `true`. Since this is an internal reason (the way our dependencies are structured), we should we still document it?

Reply:
Nah, it's fine.